uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_description/urdf/sensors.macro.xacro

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+
+  <!-- import macros file -->
+  <xacro:include filename="$(find uwrt_mars_rover_drivetrain_description)/urdf/drivetrain.macro.xacro"/>
+
+  <xacro:macro name="generate_sensor_frame" params="sensor_params name">
+    <link name="${name}">
+      <xacro:identical_visual_collision>
+        <identical_blocks>
+          <geometry>
+            <box size="${sensor_params['dim']['length']} ${sensor_params['dim']['width']} ${sensor_params['dim']['height']}"/>
+          </geometry>
+          <material name="blue">
+            <color rgba="0 0 0.8 1.0"/>
+          </material>
+        </identical_blocks>
+      </xacro:identical_visual_collision>
+      <xacro:default_inertial/>
+    </link>
+
+    <joint name="${sensor_params['parent_link']}_to_${name}" type="fixed">
+      <parent link="${sensor_params['parent_link']}"/>
+      <child link="${name}"/>
+      <origin xyz="${sensor_params['offset']['x']}     ${sensor_params['offset']['y']}     ${sensor_params['offset']['z']} " 
+              rpy="${sensor_params['offset']['r_rot']} ${sensor_params['offset']['p_rot']} ${sensor_params['offset']['y_rot']} "/>
+    </joint>
+
+  </xacro:macro>
+
+
+  <xacro:macro name="default_depth_camera_frame" params="camera_link_name">
+    <gazebo reference="${camera_link_name}">
+      <sensor name="${camera_link_name}_sensor" type="depth">
+        <visualize>true</visualize>
+        <update_rate>30.0</update_rate>
+        <camera name="camera">
+          <horizontal_fov>1.047198</horizontal_fov>
+          <image>
+            <width>640</width>
+            <height>480</height>
+            <format>R8G8B8</format>
+          </image>
+          <clip>
+            <near>0.05</near>
+            <far>3</far>
+          </clip>
+        </camera>
+        <plugin name="depth_camera_controller" filename="libgazebo_ros_camera.so">
+          <cameraName>${camera_link_name}</cameraName>
+          <imageTopicName>ir/image_raw</imageTopicName>
+          <cameraInfoTopicName>ir/camera_info</cameraInfoTopicName>
+          <depthImageTopicName>depth/image_raw</depthImageTopicName>
+          <depthImageCameraInfoTopicName>depth/camera_info</depthImageCameraInfoTopicName>
+          <pointCloudTopicName>depth/points</pointCloudTopicName>
```

Comment:
How do we ensure that the topic names do not clash? For example, if we have 2 cameras, won't all of these topics be the same? If so, we should add a namespace to these topics to prevent them from clashing if we have multiple cameras

Reply:
![Screenshot from 2023-09-18 22-25-30](https://github.com/uwrobotics/uwrt_mars_rover/assets/79111376/e309e983-a4c6-4567-bfbd-a43d21aff489)
these are the names for the topics i change it to